tools/distpartitioning/data_proc_pipeline.py

```diff
@@ -45,7 +47,10 @@ def start_local_run(params):
     #Invoke `target` function from each of the spawned process for distributed 
     #implementation
     for rank in range(params.world_size):
-        p = mp.Process(target=single_dev_init, args=(rank, params.world_size, proc_exec, params))
+        if(params.mul_files_dataset): 
```

Comment:
do you want to support both multiple file inputs and single file input?

Reply:
As agreed per our last conversation, I am removing functionality for the single-file-format support. From now on, we will only support multiple-file-format datasets !!!. Removed code in the process which will now become obsolete.